javascript/src/test/pages/SearhHistory/SearchHistory.test.js

```diff
@@ -1,11 +1,28 @@
 import React from 'react';
-import { act, render } from '@testing-library/react';
+import { render } from '@testing-library/react';
 import SearchHistory from 'main/pages/SearchHistory/SearchHistory';
+import useSWR from "swr";
+
+jest.mock("swr");
+
 
 describe('Slack Users tests', () => {
-    act(()=>{
-        test('renders without errors', () => {
+  
+        test('Member User', () => {
+            useSWR.mockReturnValue({
+      roleInfor: {
+        role: "member"
+      }
+    });
             render(<SearchHistory />);
         });
-    });   
-});
\ No newline at end of file
+        
+          test('Admin User', () => {
+            useSWR.mockReturnValue({
+      roleInfor: {
+        role: "Admin"
+      }
+    });
+        render(<SearchHistory />);
+        });
```

Comment:
These tests don't seem to be testing anything - it's just rendering the `SearchHistory` component when the user is a member and when the use is an admin. Can we instead make these tests test something, or remove them entirely? If they're not doing anything, they shouldn't be in the codebase.

Additionally, fixes to the formatting would be nice.

Also, it looks like `roleInfo` is misspelled, there's an extra `r` at the end.

Reply:
Remove useless test.